packages/node-sdk/src/client.ts

```diff
@@ -956,13 +1046,14 @@ export class BoundBucketClient {
    * @param context User/company/other context to bind to the client object
    * @returns new client bound with the additional context
    */
-  public bindClient({ user, company, other }: Context) {
+  public bindClient({ user, company, other, enableTracking }: Context) {
```

Comment:
Context has a very specific meaning: it's what you run the targeting rules on. `enableTracking` is just configuration for this new Client. Let's keep 'em separate. In other words, Context stays the way it is but `bindClient` and `getFeature`/`getFeatures` take an object `Context & {enableTracking: boolean}`